docs/guides/cases/connect-mqtt.md

```diff
@@ -38,15 +56,15 @@ kubectl apply -f examples/mqttDeviceShifu/mqtt_deploy
 In your cluster:
 
 ```
-curl deviceshifu-mqtt/mqtt_data
+curl deviceshifu-mqtt/get_topicmsg1
 ```
 
-Where `mqtt_data` is the embedded query string.
+Where `get_topicmsg1` is the embedded query string.
 
 Return from MQTT ***deviceShifu***:
 
 ```json
-{"mqtt_message":"test2333","mqtt_receive_timestamp":"2022-04-29 08:57:49.9492744 +0000 UTC m=+75.407609501"}
+{"mqtt_message":"test_topicmsg1","mqtt_receive_timestamp":"2022-04-29 08:57:49.9492744 +0000 UTC m=+75.407609501"}
```

Comment:
let's use a different `mqtt_message` value to avoid confusion between the topic and message